proposals/NNNN-initial-and-hint-text-for-keyboard.md

```diff
@@ -0,0 +1,51 @@
+# Arguments initialText and hintText for SDL Keyboard
+
+* Proposal: [SDL-NNNN](NNNN-initial-and-hint-text-for-keyboard.md)
+* Author: [MarekForys](https://github.com/mforys) (TomTom)
+* Status: **Awaiting review**
+* Impacted Platforms: [Core, Android, iOS]
+
+
+## Introduction
+
+SDl Keyboard may be started with argument initialText, but it's displayed in iOS version as blue hint/description in the input field of SDL Keyboard. In the case of Android this initial text is never displayed.
+
+There should be a possibility of using initial and hint/description text as well.
+All available documentation is here: [SDLPerformInteraction](https://smartdevicelink.com/en/docs/iOS/master/Classes/SDLPerformInteraction/).
+
+
+## Motivation
+
+Sometimes user wants to continue to input a text which partially is already filled in device's input text field, then after connecting to SDL the text field is totally empty. Another scenario is when some text is filled into SDl keyboard then we press Enter and we launch Keyboard again. Input text field is blank and previously entered text is gone.
```

Comment:
Please update first sentence to "Sometimes a user wants to continue to input text which is already partially filled in within their device's input text field, then after..."

Please update the second sentence to use "SDL" (all capitalized letters).

Reply:
Done